mmtrack/models/data_preprocessors/data_preprocessor.py

```diff
@@ -118,7 +118,8 @@ def forward(self, data: dict, training: bool = False) -> Dict:
                 imgs = [_img[:, [2, 1, 0], ...] for _img in imgs]
             if self._enable_normalize:
                 imgs = [(_img - self.mean) / self.std for _img in imgs]
-
+            # change to `float`
+            imgs = [_img.float() for _img in imgs]
```

Comment:
better to put this code in the front of  line 119 `if self._enable_normalize:`, in order to be consistent with mmengine